bounded_counter/count_min_sketch.py

```diff
@@ -2,6 +2,7 @@
 
 import numpy
 import xxhash
+from HLL import HyperLogLog
```

Comment:
This looks very tricky. Supporting a 3rd party package (especially one in C) is always a maintenance headache. Its support will be on us in reality -- can we vouch for the code / its portability / support?

How difficult is it to implement HyperLogLog "natively"?

Reply:
> Its support will be on us in reality
Yeah, I agree. In this case, it's simple enough, and seems solid.

What do you mean, 'natively'? I deliberately chose the C impl out of multiple options because it's 20x faster than an alternative pure-python [hyperloglog](https://github.com/svpcom/hyperloglog) that I've tried.

Both of them are pretty canonical too: I've checked the source and I can't imagine doing anything differently, maybe apart from choosing a different hash function (and I love this one's use of MMH)

Reply:
We could copy-paste the C code and bundle internally (if that's what you prefer), the license allows that.
In fact, that would directly give us access to the C MMH that I could use in our cythonized Count-min Sketch :)

Reply:
Yes, that's much better :)

By "natively" I meant within our control, without relying on 3rd party code and release cycles.